### PR TITLE
Bumped up the timeout for testAccessibilityFocusOnTextSemanticsProducesCorrectIosViews

### PR DESCRIPTION
**Pull request by gaaclarke**
related issue: https://github.com/flutter/flutter/issues/55778

### CODE REVIEW

**Comment by gaaclarke** [comment]
@xster I made those changes we discussed over VC.

File: testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m [hunk 1]
```diff
@@ -6,6 +6,36 @@
 
 FLUTTER_ASSERT_ARC
 
+@interface XCUIElement (ftr_waitForNonExistence)
+/// Keeps waiting until the element doesn't exist.  Returns NO if the timeout is
```
Comment (xster) [inline testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m:10]: keep waiting until the element no longe exists?

**Comment by xster** [comment]
LGTM

File: testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m (hunk 1 above)
Comment (gaaclarke) [reply testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m:10]: That sounds like it has to exist beforehand.  The implementation is agnostic to that.

**Comment by gaaclarke** [comment]
Landing before Windows Web Engine finishes, it passed previously.  I just removed some print statements since then.

**Pull request merged**